ci/jenkins/test-vmc.sh

```diff
@@ -317,8 +317,7 @@ function deliver_antrea {
     sed -i -e "s/activeFlowExportTimeout: \"5s\"/activeFlowExportTimeout: \"2s\"/g" $GIT_CHECKOUT_DIR/build/yamls/$antrea_yml
     sed -i -e "s/idleFlowExportTimeout: \"15s\"/idleFlowExportTimeout: \"1s\"/g" $GIT_CHECKOUT_DIR/build/yamls/$antrea_yml
 
-    wget -c https://raw.githubusercontent.com/antrea-io/antrea/main/build/yamls/flow-aggregator.yml -O ${GIT_CHECKOUT_DIR}/build/yamls/flow-aggregator.yml
-
+    ${GIT_CHECKOUT_DIR}/hack/generate-manifest-flow-aggregator.sh > ${GIT_CHECKOUT_DIR}/build/yamls/flow-aggregator.yml
```

Comment:
Since we only want to provide helm chart installation in Theia perhaps we can keep pulling the yaml for FA from Theia

Reply:
Do you mean "keep pulling the yaml for FA from Antrea"? The main reason I change the test code here is to have a way to make sure that our Helm chart version FA still works as expected. I expect it will be changed back to use FA from Antrea when we move FA Helm Chart to Antrea.

Reply:
Understood.